src/shared/cmpattern/cmpattern_teamdetector.cpp

```diff
@@ -416,7 +416,7 @@ void TeamDetector::findRobotsByModel(::google::protobuf::RepeatedPtrField< ::SSL
   (void)image;
   const int MaxDetections = _other_markers_max_detections;
   Marker cen; // center marker
-  Marker markers[MaxDetections];
```

Comment:
clang complains otherwise:
> variable length array of non-POD element type 'CMPattern::Marker'

POD is short for Plain Old Data. The Marker class has a constructor and thus doesn't count as POD.